test/alternator/test_return_consumedcapacity.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# Tests for the ReturnConsumedCapacity header
+
+import pytest
+from botocore.exceptions import ClientError
+from decimal import Decimal
+from test.alternator.util import random_string, random_bytes
+
+# A basic test that gets an item from a table with and without consistency
+KB = 1024
+
+def test_basic_get(test_table):
```

Comment:
Please add another separate test for a **MISSING** item (trying to read an item that doesn't even exist).
Is it accounted as 0 units? 0.5 or 1? Anything else? We need to know.

Reply:
ok, will do